extensions_admin/pulp_docker/extensions/admin/cudl.py

```diff
@@ -53,6 +53,11 @@
 OPT_ENABLE_V2 = PulpCliOption('--enable-v2', d, required=False,
                               parse_func=okaara_parsers.parse_boolean)
 
+d = _('Whitelist of Tags to download. The format of the parameter is '
+      '"<tag_name>,<tag_name>"; for example: "latest,1,"')
+OPTION_WHITELIST_TAGS = PulpCliOption('--tags', d, required=False,
```

Comment:
once provided some tags, how can we reset them null so repo would be fully synced?

api call works
``$ curl -i -H "Accept: application/json" -X PUT -k -u admin:admin -d '{"importer_config":{"tags":null}}' 'https://localhost/pulp/api/v2/repositories/v2synctest/importers/docker_importer/'
```
 not sure what we should provide to pulp-admin?

Reply:
--tags=None --tags='None' --tags=null
is not working

Reply:
That's a good point, I will a check in the update code to remove it if None is passed.